packages/surveys/src/components/general/Survey.tsx

```diff
@@ -186,6 +186,15 @@ export function Survey({
     setQuestionId(prevQuestionId);
     onActiveQuestionChange(prevQuestionId);
   };
+
+  const question = useMemo(() => {
```

Comment:
now we have a `question` and a `currentQuestion` / `currQues` that we use in different places. 
Can we somehow manage to rename `question` to `currentQuestion` and only use this in all places? Or isn't this possible? But I think we need to do something to avoid the confusion and have one object that stores the current question that is shown.